systemverilog-plugin/UhdmAst.cc

```diff
@@ -2486,6 +2486,17 @@ void UhdmAst::process_enum_typespec()
     const auto *enum_object = (const UHDM::enum_typespec *)handle->object;
     const auto *typespec = enum_object->Base_typespec();
 
+    if (current_node->str.empty()) {
+        // anonymous typespec, check if not already created
+        if (shared.anonymous_enums.find(enum_object) != shared.anonymous_enums.end()) {
```

Comment:
Something like:
```c++
        if (const auto enum_iter = shared.anonymous_enums.find(enum_object); enum_iter != shared.anonymous_enums.end()) {
        // ...
        current_node->str = enum_iter->second();
```
to avoid double lookup.